efel/cppcore/cppcore.cpp

```diff
@@ -91,23 +92,23 @@ static vector<double> PyList_to_vectordouble(PyObject* input) {
   return result_vector;
 }
 
-void PyList_from_vectordouble(vector<double> input, PyObject* output) {
-  int vector_size;
-  int index;
+static void PyList_from_vectordouble(vector<double> input, PyObject* output) {
+  size_t vector_size = input.size();
 
-  vector_size = input.size();
-  for (index = 0; index < vector_size; index++) {
-    PyList_Append(output, Py_BuildValue("f", input[index]));
+  for (size_t index = 0; index < vector_size; index++) {
+    PyObject *obj = Py_BuildValue("f", input[index]);
+    PyList_Append(output, obj);
```

Comment:
Yeah, I have a link to the line (in app1) in the commit msg, 
PyList_Append calls it here: https://github.com/python/cpython/blob/master/Objects/listobject.c#L303